sympy/assumptions/satask.py

```diff
@@ -29,6 +29,15 @@ def satask(proposition, assumptions=True, context=global_assumptions,
         return False
 
     if not can_be_true and not can_be_false:
+        from pprint import pprint
+        pprint(can_be_true)
+        pprint(can_be_false)
+        pprint('proposition')
+        pprint(proposition)
+        pprint('assumptions')
+        pprint(assumptions)
+        pprint('relevant_facts')
+        pprint(relevant_facts)
```

Comment:
This PR is far from being submitted. I just did not find a better way to discuss the changes then to send PR.